node_modules/heroku-cli-util/test/preauth.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+let Heroku = require('heroku-client');
+let nock   = require('nock');
+
+describe('preauth', function () {
+  it('makes a POST to /apps/myapp/pre-authorizations', function () {
+    let heroku = new Heroku();
```

Comment:
'let' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).